h/views/client.py

```diff
@@ -35,6 +35,12 @@ def _client_url(request):
     csp_insecure_optout=True,
     http_cache=(60 * 5, {"public": True}),
 )
+@view_config(
+    route_name="notebook_app",
+    renderer="h:templates/app.html.jinja2",
+    csp_insecure_optout=True,
```

Comment:
```suggestion
    csp_insecure_optout=True,  # This view adds its own custom Content Security Policy
```

Can you add this comment here and to the route above. Otherwise it looks like we're disabling Content Security Policy entirely for the sidebar app which is not the case. The view itself adds alternative CSP headers.